doc/man7/EVP_PKEY-EC.pod

```diff
@@ -122,6 +122,18 @@ Used for getting and setting the encoding of an EC public key. The public key
 is expected to be a point conforming to Sec. 2.3.4 of the SECG SEC 1 ("Elliptic
 Curve Cryptography") standard.
 
+=item "qx" (B<OSSL_PKEY_PARAM_EC_PUB_X>) <octet string>
+
+Used for getting the EC public key X component.
```

Comment:
Is it worth saying something about the contents?

Reply:
If people dont understand what x and y are, they probably shouldnt be using it

Reply:
What format is expected?

Reply:
Its raw byte data for an x point

Reply:
Having it return compressed data without the format would be silly. Do you want it to explicitly say uncompressed?

Reply:
It's an integer, what byte order?  Blocked into larger units?

_which is an integer stored in big endian format_ or similar.

Reply:
Do we do this for any other key fields? Why pick on this one.

Reply:
Are you saying because it an octet string..

Reply:
Guess what - it isnt an octet string - the gettableis wrong :)

Reply:
Yeah, if it were a bignum, it would be fine as is.

Reply:
I am wondering if there is some way to check if all the gettable tables match the get values.. There have been a few of these mismatches.. Some gettables were even in the wrong tables (e.g. in the get rather than the get_ctx)